### DISCUSSION TITLE
Reserved stock and custom orders table

### DISCUSSION THREAD

**Discussion by Screenfeed** [woocommerce/woocommerce#36153] [WooCommerce HPOS] [answered]
Hello there 👋

Looking at `ReserveStock::get_query_for_reserved_stock()`, the query is tied to the posts table and I can't find anything modifying/replacing it when the COT is used.
https://github.com/woocommerce/woocommerce/blob/60f2ced82ba798e74402b7baabe9ca3bcd60a2d5/plugins/woocommerce/src/Checkout/Helpers/ReserveStock.php#L203-L228

So here is what I'd like to know: is it an oversight? Or maybe a work in progress? Or not needed when the COT is used (seems unlikely to me)?

If I ask this, it's because I'm one of the developers of Polylang and we're filtering this query. We're adapting our code to the COT and I wanted to make sure we're not missing something about this part.

Thanks.
Greg

**Comment by vedanshujain** [comment, marked as answer]
Seems like an oversight, I will add an issue to track this. Thanks for your report!

**Reply by Screenfeed** [reply]
Thank you 👍
